src/main/java/com/github/monster860/fastdmm/objtree/ObjectTreeParser.java

```diff
@@ -243,16 +92,16 @@ public void doParse(File file, boolean isMainFile) throws IOException
 			}
 		}
 		br.close();
-		
+
 		ArrayList<String> pathTree = new ArrayList<>();
-		
+
 		int currentInclude = 0;
-		
+
 		JProgressBar dpb = null;
 		JDialog dlg = null;
 		JLabel lbl = null;
 		if(isMainFile) {
-			final JDialog tdlg = new JDialog(modalParent, "Object Tree Generation", modalParent == null ? false : true);
+			final JDialog tdlg = new JDialog(modalParent, "Object Tree Generation", modalParent != null);
```

Comment:
I didn't actually make that variable I just removed a redundant ternary operator.